flang/docs/OpenMPStandardsSupport.md

```diff
@@ -0,0 +1,62 @@
+<!--===- docs/FortranStandardsSupport.md
+
+   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+   See https://llvm.org/LICENSE.txt for license information.
+   SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+-->
+
+# Flang OpenMP Standards Support
+
+```{contents}
+---
+local:
+---
+```
+
+This document summarizes OpenMP standards support in Flang. The information is only provided as a guideline. The
```

Comment:
```suggestion
This document summarizes the supported features of the OpenMP API in Flang. The information is only provided as a guideline. The
```

(OpenMP is not a standard in the true sense, like with the ISO standards.)

Reply:
Would it be OK to go with OpenMP specification?
`This document summarizes the supported features of the OpenMP specification in Flang`

Reply:
Done.